### PR TITLE
Feature/document fields

### PR DESCRIPTION
**Pull request by LindsayYoung**
Adds documentation to most of the fields. 

I am trying to make a good base that puts Paul and Jeff in a better place to improve it with more specifics. 

It mostly uses existing work for documenting arguments and I added a few straight forward things. 

There is also a function that makes some generic definitions from the variable name. Not sure how helpful this is, but it makes the documentation look a lot more filled out. Happy to get feedback on that approach. 

I am getting a swagger integration error from py.test, but it all looks like it is all working to me.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
First step of #1416 
Need to add the definitions to the manifest too.

**Comment by jmcarp** [comment]
The failing swagger tests means that we're generating invalid swagger. Here's the actual error message:

screen shot 2016-01-25 at 10 12 24 am

File: webservices/docs.py [hunk 1]
```diff
@@ -469,3 +470,187 @@
...
+
+def add_period(var):
+    return var + ' total for the reporting period'
+
+
+def add_ytd(var):
+    return var + ' total for the year to date'
+
+
+def make_generic_def(var):
```
Comment (jmcarp) [inline webservices/docs.py]: I'm not sure I see the point of this helper. It looks like we're more or less copying the column name into the documentation field, modulo some spacing cleanup. But we aren't actually adding any information to the documentation--we're showing the same information twice. Imagine this in the download manifest:

aggregate_amount_personal_contributions_general: Aggregate amount personal contributions general

Comment (LindsayYoung) [inline webservices/docs.py]: My thinking was that it was better than nothing, but I can be persuaded to remove it.

**Comment by LindsayYoung** [comment]
I need to fix the swagger error and resolve conflicts, but in the meantime, I wanted to flag @emileighoutlaw on the additional language for the documentation

**Comment by emileighoutlaw** [comment]
Thanks thanks! Taking a look now.

**Comment by emileighoutlaw** [comment]
This is rad, @LindsayYoung! I just suggested a handful of tiny copyedits.

File: webservices/docs.py [hunk 2]
```diff
@@ -469,6 +476,14 @@
 - The communication is distributed within 60 days prior to a general election or 30 days prior to a primary election to federal office.
 '''
 
+# merge question????????????????
```
Comment (LindsayYoung) [inline webservices/docs.py]: @emileighoutlaw in merging the docs, I noticed we have 2 versions of electioneering, I like the first one because it is more specific, with days and says federal candidate instead of House and Senate, (because I see presidential electioneering so only house or senate doesn't make sense to me). Wondered if this came up for you before.

Data source:
http://www.fec.gov/finance/disclosure/electioneering.shtml

Comment (emileighoutlaw) [inline webservices/docs.py]: Oh! I hadn't seen that before, but I think you're definitely right!

File: webservices/docs.py [hunk 3]
```diff
@@ -481,8 +482,14 @@
     - U Unregistered Committee Notice
 '''
 DATES_TAG = '''
-Regular reporting deadlines and election dates.
+Reporting deadlines, election dates FEC meetings, events etc.
 '''
+
+CALANDAR_DATES = '''
```
Comment (jmcarp) [inline webservices/docs.py]: CALENDAR

File: webservices/tasks/download.py [hunk 4]
```diff
@@ -168,7 +168,7 @@ def make_manifest(resource, row_count, path):
         fp.write('Time: {}\n'.format(resource['timestamp']))
         fp.write('Resource: {}\n'.format(resource['path']))
         fp.write('*Count: {}\n'.format(row_count))
-        fp.write('Filters:\n\n')
+        fp.write('Fields:\n\n')
```
Comment (jmcarp) [inline webservices/tasks/download.py]: Why rename this? Did we add fields?

Comment (LindsayYoung) [inline webservices/tasks/download.py]: I was in the process of adding them, I wanted that to be a separate pr, I thought I rolled that back.

File: webservices/docs.py (hunk 3 above)
Comment (LindsayYoung) [inline webservices/docs.py]: Thanks! I have a serious issue with spelling that right, and It apparently doesn't matter how many times I have had to type that in the last few months.
